diff --git a/node_modules/vite-express/dist/main.js b/node_modules/vite-express/dist/main.js
index f1f8eff..39cf5db 100644
--- a/node_modules/vite-express/dist/main.js
+++ b/node_modules/vite-express/dist/main.js
@@ -258,8 +258,7 @@ function startServer(server) {
         var _a, _b;
         const { createServer, mergeConfig } = yield import('vite');
         const config = yield getViteConfig();
-        const isUsingViteResolvedConfig = Object.entries(config).length > 3;
-        const vite = yield createServer(mergeConfig(isUsingViteResolvedConfig ? {} : config, {
+        const vite = yield createServer(mergeConfig(config, {
             configFile: Config.viteConfigFile,
             clearScreen: false,
             appType: "custom",
